crates/pessimistic-proof-test-suite/src/bin/ppgen.rs

```diff
@@ -25,7 +25,7 @@ struct PPGenArgs {
     n_exits: usize,
 
     /// The number of imported bridge exits.
-    #[clap(long, default_value = "0")]
+    #[clap(long, default_value = "10")]
```

Comment:
I'm fine with this. An alternative would be to remove the default so the user has to specify the exit count explicitly. While it's not much of an issue for this tool, the default of 10 feels a bit arbitrary.